package.json

```diff
@@ -233,6 +233,7 @@
     "yn": "^3.0.0"
   },
   "dependencies": {
+    "@onfido/active-video-capture": "file:../active-video-capture",
```

Comment:
where is this coming from?

Reply:
makes sense, I figured that was what was happening but wanted to double check in case there was some magic thing I wasn't aware of 👌

Reply:
Updated to use the published package.